DocSum/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -52,6 +52,18 @@ cd intel/cpu/xeon/
 docker compose up -d
 ```
 
+To deploy DocSum services with remote endpoints, set the required environment variables mentioned below and run the 'compose_remote.yaml' file.
+
+**Note**: Set LLM_ENDPOINT variable value to "https://api.inference.denvrdata.com" when the remote endpoint to access is "https://api.inference.denvrdata.com/v1/chat/completions"
+
+```bash
+export LLM_ENDPOINT=<endpoint-url>
+export LLM_MODEL_ID=<model-id>
```

Comment:
Better to mention how users to set LLM_MODEL_ID correctly by getting the supported model list